internal/handler/msg_getmore.go

```diff
@@ -226,7 +228,12 @@ func (h *Handler) MsgGetMore(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		}
 
 	case cursor.TailableAwait:
-		panic("not supported yet")
+		if nextBatch.Len() == 0 {
```

Comment:
It looks a bit sus that it is not `if next batch.Len() < int(batchSize)`. Should it be? Or we could simplify the code above to use `if nextBatch.Len() == 0`?

Please look into it in the next PR. Maybe there is a way to test this.

Ack and resolve.